modules/dropdown/main/index.coffee

```diff
@@ -123,6 +128,20 @@ class Dropdown extends hx.EventEmitter
       selection.on('mouseover', 'hx.dropdown', onmouseover)
       selection.on('mouseout', 'hx.dropdown', onmouseout)
 
+  dropdownContent: (dropdownContent) ->
+    if arguments.length
+      setupDropdown = dropdownContentToSetupDropdown dropdownContent
+      @_ = hx.shallowMerge @_, {
```

Comment:
Could this rather set the fields directly on `@_` rather than creating a new copy of it and reassigning that?